### ISSUE TITLE
Blows up if `pkg.bin` is a string

### ISSUE THREAD

**Issue by shakyShane** [sindresorhus/meow#11]
- this line https://github.com/sindresorhus/meow/blob/master/index.js#L35
- introduced in https://github.com/sindresorhus/meow/commit/d1a418c5aa800b91f89f8ab6ffd637235de90bc9

... is troublesome since the ternary reports true on a string, but then the interpreter spews when trying to call `Object.keys` on it.

I would've provided tests + a fix but your test suit does not (to my knowledge) allow the input of arbitrary pkg objects

use case, to wrap up

```json 
"bin": "bin/browser-sync.js",
```

```shell
/Users/shakyshane/sites/oss/browser-sync/node_modules/meow/index.js:36
        process.title = pkg.bin ? Object.keys(pkg.bin)[0] : pkg.name;
                                         ^
TypeError: Object.keys called on non-object
    at Function.keys (native)
    at module.exports (/Users/shakyshane/sites/oss/browser-sync/node_modules/meow/index.js:36:35)
    at Object.<anonymous> (/Users/shakyshane/sites/oss/browser-sync/bin/browser-sync.js:14:11)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3

```

**Comment by sindresorhus** [comment]
Ah, sorry about that. I use `read-pkg-up` which normalizes the package data, but I forgot to run the normalization when the package or package path is specified.

Btw, you no longer have to manually specify the package.json path as `meow` will search up the dir tree for it ;)

**Comment by shakyShane** [comment]
> Fixed in 3.4.2 

:beer: 

> you no longer have to manually specify the package.json path

:beers:
